readme.md

```diff
@@ -147,6 +147,13 @@ Values: `black` `red` `green` `yellow` `blue` `magenta` `cyan` `white`
 
 Color of the background.
 
+##### align
+
+Type: `string`<br>
+Values: `center` `right`
```

Comment:
@sindresorhus Any concern that users would expect `align: 'left'` to "undo" left-padding? Because currently `ansi-align` doesn't do that. 😅